src/components/Counter/counter.scss

```
@@ -0,0 +1,5 @@
+.TodoList--counter{

```

```suggestion
.counter {
```

You don't need to reference to the `.todoList` since it's a different component. You can use the name of the component as base of the CSS class name